### PR TITLE
	Implement possibility to display/hide home block filters in admin

### PR DESCRIPTION
**Pull request by HeahDude**

### CODE REVIEW

File: app/migrations/Version20170601095909.php [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+namespace Migrations;
+
+use Doctrine\DBAL\Migrations\AbstractMigration;
+use Doctrine\DBAL\Schema\Schema;
+
+class Version20170601095909 extends AbstractMigration
+{
+    public function up(Schema $schema)
+    {
+        $this->addSql('ALTER TABLE committee_feed_item DROP published');
```
Comment (tgalopin) [inline app/migrations/Version20170601095909.php]: This seems wrong

File: src/AppBundle/Admin/HomeBlockAdmin.php [hunk 2]
```diff
@@ -64,6 +64,9 @@ protected function configureFormFields(FormMapper $formMapper)
             ])
             ->add('link', null, [
                 'label' => 'Cible du lien',
+            ])
+            ->add('link', null, [
```
Comment (tgalopin) [inline src/AppBundle/Admin/HomeBlockAdmin.php]: This seems wrong

File: app/migrations/Version20170601095909.php [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+        $this->addSql('ALTER TABLE home_blocks ADD display_filter TINYINT(1) DEFAULT \'1\' NOT NULL');
+    }
+
+    public function down(Schema $schema)
+    {
+        $this->addSql('ALTER TABLE committee_feed_item ADD published TINYINT(1) DEFAULT \'1\' NOT NULL');
```
Comment (tgalopin) [inline app/migrations/Version20170601095909.php]: This seems wrong

File: app/migrations/Version20170601095909.php (hunk 1 above)
Comment (HeahDude) [inline app/migrations/Version20170601095909.php]: Fixed.

File: app/migrations/Version20170601095909.php (hunk 3 above)
Comment (HeahDude) [inline app/migrations/Version20170601095909.php]: Fixed.

File: src/AppBundle/Admin/HomeBlockAdmin.php (hunk 2 above)
Comment (HeahDude) [inline src/AppBundle/Admin/HomeBlockAdmin.php]: Fixed.

**Pull request merged**